Uchu.StandardScripts/General/StatPickups.cs

```diff
@@ -8,63 +10,33 @@ public class StatPickups : NativeScript
     {
         public override Task LoadAsync()
         {
+            var pickups = new Dictionary<int, int>()
```

Comment:
These don't need to be hardcoded, they are defined in the CDClient: the objects have an attached skill with `castOnType = 2` (table ObjectSkills).
There are actually a few more objects like this with a `castOnType = 2` skill: 7680, 8200 and 8208 (powerups for racing).